### DISCUSSION TITLE
Missing changelog

### DISCUSSION THREAD

**Discussion by pydanny** [gnat/surreal#43] [Ideas]

**Comment by gnat** [comment]
Gonna convert this to discussion.

**Comment by gnat** [comment]
Thanks for the feedback! Might happen at some point just not sure if the added friction is worth it currently.

To be fair: This does exist chronologically as history messages- it's just not summarised.

Surreal is a small project and backwards compatibility is of highest priority.

**Comment by ethanpil** [comment]
I too was hoping for a changelog. Perhaps one of these tools or or similar would reduce some of the friction fpr @gnat ?
https://github.com/requarks/changelog-action
https://git-cliff.org/
https://github.com/conventional-changelog/conventional-changelog

**Comment by gnat** [comment]
Totally agreed in retrospect. Thank you for the reminder @ethanpil 

You can find the new change log at the end of the README.md: https://github.com/gnat/surreal#-change-log

Bullet list of changes will also be available on the Releases page moving forward.
